src/terminal/adapter/lib/adapter.vcxproj

```diff
@@ -14,7 +14,6 @@
     <ClCompile Include="..\DispatchCommon.cpp" />
     <ClCompile Include="..\InteractDispatch.cpp" />
     <ClCompile Include="..\adaptDispatchGraphics.cpp" />
-    <ClCompile Include="..\MouseInput.cpp" />

```

Comment:
you've updated a vcxproj file: you have to update the `sources` file in the same directory